msteams-platform/get-started/get-started-overview.md

```diff
@@ -1,64 +1,80 @@
+description: In this module, learn how to build your first Microsoft Teams app based on language and developement environment, understand app capabilities, SDKs.
 ms.localizationpriority: high
 ms.topic: reference
 ---
 # Get started
 
-Welcome to Get started for building and deploying customized apps for Microsoft Teams!
+Let's get started with building apps for Microsoft Teams and beyond!
 
-Walk through the steps to build a basic, real-world Teams app. The Get started also introduces you to common tools, fundamental concepts, and more advanced features.
+Experience the Teams app with GitHub Codespaces or walk through the steps to build a basic, real-world Teams app. The Get started also introduces you to common tools, fundamental concepts, and more advanced functionalities.
 
 Here's an idea of what you'll learn:
 
-- Get up and running quickly with the Microsoft Teams Toolkit (a Visual Studio Code extension).
-- Get experience with the Toolkit and SDKs.
-- Configure and build different types of Teams apps.
+* Get up and running quickly with the Microsoft Teams Toolkit.
+* Build tab, scenario-based apps, and message extension with GitHub Codespaces or step-by-step guide.
+* Learn about tutorials and code samples available for building your Teams app.
+* Learn about various tools and SDKs available for building your Teams app.
 
-Let's take a quick glance at the build environment options you can choose from, and the road-map to build and deploy a Teams app.
+## Build your first Teams app
+
+In this section, learn more how to build Teams app with different capabilities, such as tab, scenario-based bot app, and message extension.
+
+1. Select the type of Teams app that you want to build:
+
+   * **Build a basic Teams tab app**: You can get started with building a Teams app with a basic tab app.
+
+   * **Build a scenario-based Teams bot app**: You can build a Teams app that is suitable for a particular scenario or use-case. For example, a notification bot is suitable to send notifications but not to chat.
+
+   * **Build message extension Teams app**: You can build a search based message extension Teams app.
 
-:::image type="content" source="../assets/images/get-started/gs-build-options.png" alt-text="Illustration showing basic steps to build and deploy a Teams app":::
+2. Choose your path to build your selected app:
 
+   * **GitHub Codespaces**: Allows you to run, test, and debug your app in a development environment without the need for any of the setup.
+   * **Step by step guide**: Allows you to create a Teams app from scratch with all development environment setup.
+
+   :::image type="content" border="false" source="../assets/images/get-started/codescpaces-vs-manual.png" alt-text="Screenshot shows your choices to build Teams app with codespaces and step-by-step guide.":::
```

Comment:
Can you add the image magnification 🔍  (is it called "Lightbox"?) for this image too? The text in the image is too small to read when it is sized down in the doc. Thank you!

Reply:
Added lightbox